src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -24,27 +24,27 @@ export default function CSVFileImport({ url, title }: CSVFileImportProps) {
   };
 
   const uploadFile = async () => {
-    console.log("uploadFile to", url);
     if (!file) {
       console.log('File doesnt exist');
       return;
     }
-    // Get the presigned URL
+
     const response = await axios({
-      method: "GET",
+      method: 'GET',
       url,
       params: {
-        name: encodeURIComponent(file.name),
+        name: encodeURIComponent(file && file.name),
+      },
+      headers: {
+        Authorization: `Basic ${localStorage.getItem("authorization_token")}`,
```

Comment:
You should encode the token before sending it to import service.
<img width="1173" alt="image" src="https://user-images.githubusercontent.com/15226768/202479706-5ca1ad69-83b4-4d40-8060-f78060b00a88.png">